### DISCUSSION TITLE
23.10.25 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#44] [Daily Scrum]
.

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 참회의 코드리뷰...
* 내일은 어떤 일을 하나요?
  * StreamingHistory용 DataSource 선정
  * PR

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 면접 준비,,🥲
    - 코드 리뷰 진행 중
- 내일은 어떤 일을 하나요?
    - 코드 리뷰 마치기 (회의 끝나고)

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 코드 PR
    - 우재님 코드 리뷰
    - 공통 예외 처리 추가
- 내일은 어떤 일을 하나요?
    - 코드 리뷰 반영
    - 플레이리스트 노래 추가

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - PR 수정 & 코드 리뷰
  - 비동기 학습 및 적용
- 내일은 어떤 일을 하나요?
  - 비동기 적용 테스트 코드 작성
  - 유저와 코드 병합?
